### PR TITLE
FossGraph. Added a new view for project problem info

### LINKED ISSUE #2082: FossGraph. Add a new view for project problem info

### PR DESCRIPTION
**Pull request by Cheshiriks**
Added a new view for project problem info

Closes #2082 

Снимок экрана 2023-04-14 в 18 04 48

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectProblemView.kt [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+import react.router.dom.Link
+import react.useState
+
+val projectProblem: FC<ProjectProblemViewProps> = FC {props ->
+
+    val (projectProblem, setProjectProblem) = useState(ProjectProblemDto.empty)
+
+    useRequest {
+        val projectProblemNew: ProjectProblemDto = get(
+            url = "$apiUrl/projects/problem/get/by-id?id=${props.projectProblemId}",
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectProblemView.kt:32]: Suggestion to use `get` that receives query params built by `jso`. You can find implementation in the same file as your `get`

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectProblemView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectProblemView.kt]: Fixed

**Review by 0x6675636b796f75676974687562 — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectSecurityMenu.kt [hunk 2]
```diff
@@ -40,7 +40,7 @@ private val problemsTable: FC<TableProps<ProjectProblemDto>> = tableComponent(
                 Fragment.create {
                     td {
                         Link {
-                            to = "/${cellContext.row.original.name}"
+                            to = "/project/${cellContext.row.original.organizationName}/${cellContext.row.original.projectName}/security/problems/${cellContext.row.original.id}"
```
Comment (0x6675636b796f75676974687562) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectSecurityMenu.kt:43]: How about extracting `cellContext.row.original` into a variable?

Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectSecurityMenu.kt]: Agree. Changed it

**Review by nulls — APPROVED** [review]

**Pull request merged**